dagobah/core/core.py

```diff
@@ -252,6 +256,36 @@ def add_task_to_job(self, job_or_job_name, task_command, task_name=None,
         job.add_task(task_command, task_name, **kwargs)
         job.commit()
 
+
+    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None):
+        """ Add a task to a job owned by the Dagobah instance. """
+
+        if isinstance(job_or_job_name, Job):
+            job = job_or_job_name
+        else:
+            job = self.get_job(job_or_job_name)
+
```

Comment:
Next commit: abstracting to a `_resolve_job` function that will give you the job object, whether you pass it the ID or the job itsself.